pac4j-core/src/main/java/org/pac4j/core/credentials/extractor/HeaderExtractor.java

```diff
@@ -36,7 +36,7 @@ public TokenCredentials extract(WebContext context) throws HttpAction, Credentia
             throw new CredentialsException("Wrong prefix for header: " + this.headerName);
         }
 
-        final String headerWithoutPrefix = header.substring(this.prefixHeader.length());
+        final String headerWithoutPrefix = header.substring(this.prefixHeader.length()).trim();
```

Comment:
As I said in another comment, let's make the trimming an option driven by a boolean property, something like `trimValue`.

Reply:
Sure, will make that change.